suites/squid/nvmeof/tier-3_nvmeof_4gwgroup_8gw_ha.yaml

```diff
@@ -0,0 +1,443 @@
+                    size: 2G
+                    lb_group: node14
+                  - count: 2
+                    size: 2G
+                    lb_group: node15
+                  - count: 2
+                    size: 2G
+                    lb_group: node16
+                  - count: 2
+                    size: 2G
+                    lb_group: node17
+              - nqn: nqn.2016-06.io.spdk:cnode6
+                listener_port: 4420
+                listeners: [node14, node15, node16, node17]
+                allow_host: "*"
+                bdevs:
+                  - count: 2
+                    size: 2G
+                    lb_group: node14
+                  - count: 2
+                    size: 2G
+                    lb_group: node15
+                  - count: 2
+                    size: 2G
+                    lb_group: node16
+                  - count: 2
+                    size: 2G
+                    lb_group: node17
+            fault-injection-methods:                # Failure induction
+              - tool: systemctl
+                nodes: node14
+          - gw_group: group4
+            gw_nodes:
+              - node18
+              - node19
+              - node20
+              - node21
+            subsystems:
+              - nqn: nqn.2016-06.io.spdk:cnode7
+                listener_port: 4420
+                listeners: [node18, node19, node20, node21]
+                allow_host: "*"
+                bdevs:
+                  - count: 2
+                    size: 2G
+                    lb_group: node18
+                  - count: 2
+                    size: 2G
+                    lb_group: node19
+                  - count: 2
+                    size: 2G
+                    lb_group: node20
+                  - count: 2
+                    size: 2G
+                    lb_group: node21
+              - nqn: nqn.2016-06.io.spdk:cnode8
+                listener_port: 4420
+                listeners: [node18, node19, node20,node21]
+                allow_host: "*"
+                bdevs:
+                  - count: 2
+                    size: 2G
+                    lb_group: node18
+                  - count: 2
+                    size: 2G
+                    lb_group: node19
+                  - count: 2
+                    size: 2G
+                    lb_group: node20
+                  - count: 2
+                    size: 2G
+                    lb_group: node21
+            fault-injection-methods:                # Failure induction
+              - tool: systemctl
+                nodes: node18
+          - gw_group: group5
+            gw_nodes:
+              - node22
+              - node23
+              - node24
+              - node25
+            subsystems:
+              - nqn: nqn.2016-06.io.spdk:cnode9
+                listener_port: 4420
+                listeners: [node22, node23, node24, node25]
+                allow_host: "*"
+                bdevs:
```

Comment:
I think we need to bring another idea to tackle bdevs and probably other config settings.

Reply:
Will try to take it up in another PR